src/util/Util.ts

```diff
@@ -60,19 +60,24 @@ export const matchingMediaType = (mediaA: string, mediaB: string): boolean => {
 
 /**
  * Pipes one stream into another.
- * Makes sure an error of the first stream gets passed to the second.
+ * In case of an error in the first stream the second one will be destroyed with the given error.
  * @param readable - Initial readable stream.
  * @param destination - The destination for writing data.
  * @param mapError - Optional function that takes the error and converts it to a new error.
  *
- * @returns The destination stream.
+ * @returns The pipeline stream.
  */
 export const pipeStreamsAndErrors = <T extends Writable>(readable: NodeJS.ReadableStream, destination: T,
```

Comment:
Just a comment: would be neat if we'd move this to a dedicated library.
I'm applying this pattern myself a lot while coding as well, so it'd be good to have this abstracted.

Reply:
Is putting 5 lines of code in a separate npm package not one of those things that is being discouraged of doing in the npm repository? 😄